datadog_checks_dev/datadog_checks/dev/tooling/testing.py

```diff
@@ -189,14 +190,15 @@ def construct_pytest_options(
         pytest_options += ' --benchmark-skip'
 
     if junit:
+        test_group = 'e2e' if e2e else 'unit'
```

Comment:
How would this handle benchmarks?

Reply:
Benchmark would be "unit" since, it's run via `ddev test` and not `ddev env test`.

Reply:
`ddev test` doesn't imply unit tests though